sdk/storage/storage-blob/test/browser/highlevel.browser.spec.ts

```diff
@@ -93,6 +95,7 @@ describe("Highlevel", () => {
     let eventTriggered = false;
     const aborter = new AbortController();
 
+    /* eslint no-empty: ["error", { "allowEmptyCatch": true }] */
```

Comment:
What is the reason for ignoring the error here? Shouldnt failure to upload data also fail the test?